src/fluxgui/fluxapp.py

```diff
@@ -58,31 +63,31 @@ class Indicator(object):
     def __init__(self, fluxgui, xflux_controller):
         self.fluxgui = fluxgui
         self.xflux_controller = xflux_controller
-        self.indicator = appindicator.Indicator(
-          "fluxgui-indicator",
-          "fluxgui",
-          appindicator.CATEGORY_APPLICATION_STATUS)
+        self.indicator = appindicator.Indicator.new(
+            "fluxgui-indicator",
+            "fluxgui",
+            appindicator.IndicatorCategory.APPLICATION_STATUS)
 
         self.setup_indicator()
 
     def setup_indicator(self):
-        self.indicator.set_status(appindicator.STATUS_ACTIVE)
+        self.indicator.set_status(appindicator.IndicatorStatus.ACTIVE)
         self.indicator.set_icon('fluxgui-panel')
         self.indicator.set_menu(self.create_menu())
 
     def create_menu(self):
         menu = gtk.Menu()
 
-        self.add_menu_item("_Pause f.lux", self._toggle_pause,
-                menu, MenuItem=gtk.CheckMenuItem)
-        self.add_menu_item("Prefere_nces", self._open_preferences, menu)
+        self.add_menu_item("Pause f.lux", self._toggle_pause,
+                           menu, MenuItem=gtk.CheckMenuItem)
+        self.add_menu_item("Preferences", self._open_preferences, menu)
```

Comment:
And again, unnecessary whitespace changes.

Reply:
> Why remove these underscores? They define shortcut keys.

Apparently this feature has changed in gtk+ 3, if I keep the underscores it will appear as part of the string in the menu and sorry about the whitespaces

Reply:
Thanks for explaining.